articles/blockchain-consensus-protocols/index.md

```diff
@@ -0,0 +1,83 @@
+
+In proof of stake consensus, a validator is picked and assigned a block. The miner has to allocate a certain part of his cryptocurrency to start validating. If the miner succeeds in invalidating the transaction, then the award is the stake they had pledged initially, along with certain transaction fees. This is a way to penalize bad behavior and incentivize good behavior.
+
+##### Properties
+
+- The validators are picked according to their economic stake in the network
+- The objective is to avoid centralization of mining centers and provide a chance to validate to all the miners
+- It is environmentally friendly as there is no computational puzzle to be solved
+- Special hardware for mining is not required
+
+
+#### Proof of Space
+
+Proof of space, also known as PoSpace, is a network consensus protocol similar to the Proof of Work consensus protocol. Instead of the computational resources, PoSpace uses disk storage to validate transactions.
+
+PoSpace consumes disk space and incentivizes miners with the most considerable disk space allocated to a block. PoSpace is implemented using the hard-to-pebble graphs. 
+
+All the feasible solutions to the problem are generated randomly, called plots. These plots are stored on the disks and solved using an algorithm called Shabal's algorithm. Once the solutions are computed, the miners compare their solutions, and the solution with the best time and space complexity is rewarded with the next block.
+
+#### Applications
+- Burstcoin is a decentralized cryptocurrency built using PoSpace. The value proposition is to aid payment systems. 
+- SpaceMint is another example in its early conceptual stages.
+- 
+#### Proof Elapsed Time
+
+Proof of Elapsed time is a network consensus protocol developed by Intel Corporation. The algorithm is predominantly used in permissioned blockchain ledgers. The hardware used in PoET is specially designed for this protocol. For example, Intel Software Guarded Extension (SGX) is used in networks using PoET. 
+
+This consensus protocol is used to allocate blocks to miners on the network. In permissioned blockchain systems, the miners' identity is determined before they are allowed access into the network. Therefore, anonymity is not achieved in this protocol. 
+
+Each node in the network is assigned a random waiting time. The first node to complete the randomly chosen time period validates the new block. The specialized hardware puts the processor to sleep during the wait time—this repeats over all the blocks in the network. 
+
+##### Applications
+
+- IBMs Hyperledger Sawtooth supports PoET mechanism for custom blockchain applications development.
+
```

Comment:
It would also be a great addition if you could discuss disadvantages of all the consensus algorithms. I know that you've mentioned it in places. But it would be great if you could add a heading, "Disadvantages" and then mention it there.

Reply:
Are you referring to a disadvantages section under each protocol?

Reply:
Let me know. I have mentioned the disadvantages under each protocol without specifying them explicitly. I will change it upon your directive.

Reply:
@lalith1403, It would be more structured to have a separate section for disadvantages. And it would definitely improve readability :)

Reply:
Let me work on it. Will ping you once done.

Reply:
Sure thing!